contrib/libspectrum/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "libspectrum"
+pkgver = "1.5.0"
+pkgrel = 0
+build_style = "configure"
+make_cmd = "gmake"
+make_build_env = {"MAKE": "gmake"}
```

Comment:
this should not be necessary (MAKE should be set by gmake itself within the invocation)